src/components/Head.js

```diff
@@ -0,0 +1,46 @@
+import NextHead from 'next/head';
+import { useRouter } from 'next/router';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Head = ({
+  title, description, ogDescription, ogImage, ogTitle,
+}) => {
+  const router = useRouter();
+  return (
+    <NextHead>
+      <title>
+        {title}
+        {' '}
+        | YSM
+      </title>
+      <meta name="description" content={description} />
+      <meta property="og:url" content={`${process.env.BASE_URL} ${router.pathname}`} />
```

Comment:
Is the space between `process.env.BASE_URL` and `router.pathname` expected/needed?

Reply:
Removed